pnpm-lock.yaml

```diff
@@ -1686,9 +1686,6 @@ importers:
 
   packages/unenv-preset:
     dependencies:
-      unenv:
-        specifier: 2.0.0-rc.1
-        version: 2.0.0-rc.1
```

Comment:
What's happening here?

Reply:
`pnpm link` to test the dev version as the new structure has not been released to npm.
Will need to be updated after the release and then remove draft from the PR